src/components/Welcome.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Welcome({ items }) {
+  //if(!items){
+  return (
+    <div>
+      <h1 className="font-Karma text-black text-center text-4xl my-4">
+        Welcome to ~name of app here~
```

Comment:
Maybe we could say "Welcome to your shopping list 'met fest hogan'.(you can share your list name with friends)" 
So far our user doesn't know a shopping list name.
what are your thoughts?